extension/prometheus/prometheus_test.go

```diff
@@ -0,0 +1,34 @@
+// +build unit
+
+package prometheus
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/hellofresh/goengine/driver/sql"
+)
+
+func TestPrometheusMetrics(t *testing.T) {
```

Comment:
Please add some tests where for example `FinishNotificationProcessing` is called before anything else etc,